cdap-docs/hydrator-manual/source/developing-plugins/creating-a-plugin.rst

```diff
@@ -24,10 +26,31 @@ extend the ``Action`` class. Only one method is required to be implemented::
 - ``configurePipeline()``: Used to create any streams or datasets or perform any validation
   on the application configuration required by this plugin.
 
+Post-run Action Plugin
+======================
+A ``PostAction`` plugin runs arbitrary logic at the end of a pipeline run.
```

Comment:
What about this instead:
```
A ``PostAction`` plugin runs arbitrary logic after the end of a pipeline run. It can be set to execute based on
whether the run completed successfully, if it failed, or in either case.
```
I suggest a blank line before the start of the next sentence to help highlight it.